shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -545,8 +546,10 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
       // For platform views we delegate the node creation to the accessibility view embedder.
       View embeddedView =
           platformViewsAccessibilityDelegate.getPlatformViewById(semanticsNode.platformViewId);
-      Rect bounds = semanticsNode.getGlobalRect();
-      return accessibilityViewEmbedder.getRootNode(embeddedView, semanticsNode.id, bounds);
+      Point offset =
+          platformViewsAccessibilityDelegate.getPlatformViewWindowOffset(
```

Comment:
I'm not sure I'm following this correctly - for VDs the view is sized to match the display and it's always positioned at (0,0) within the VD, won't it mean that this will always return (0,0) for VDs? will the translation be still correct for VDs?

Reply:
Discussed offline. This is the solution: https://github.com/flutter/engine/pull/21719/commits/eec40d1ff968103f3fae49ce6b661008ef851495

This works very well. TalkBack works the same way as in a vanilla Android app (without using Flutter).

Reply:
Done.